Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN GO111MODULE=on go build --mod=vendor cmd/corednsmonitor/corednsmonitor.go
 RUN GO111MODULE=on go build --mod=vendor cmd/monitor/monitor.go
 RUN GO111MODULE=on go build --mod=vendor cmd/unicastipserver/unicastipserver.go
 
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM registry.svc.ci.openshift.org/ocp/4.6:base
```

Comment:
This is breaking my local build. Evidently the 4.6 base image is still RHEL 7-based.

Reply:
This has since changed to RHEL 8.